docs/reST/ref/image.rst

```diff
@@ -184,6 +184,16 @@ following formats.
 
    .. ## pygame.image.tostring ##
 
+.. function:: tobytes
+
+   | :sl:`transfer image to string buffer`
+   | :sg:`tobytes(Surface, format, flipped=False) -> bytes`
+
+   since version 2.1.3 it is recommended that `tobytes` is used instead of `tostring` 
```

Comment:
Needs to mention that this is an alias of `tostring`.
Also needs the `.. versionadded::` tag